apps/src/netsim/NetSimLogBrowser.jsx

```diff
@@ -63,17 +62,11 @@ const NetSimLogBrowser = React.createClass({
       }
     }
     return header;
-  },
+  }
 
-  getInitialState() {
-    return {
-      currentSentByFilter: 'none'
-    };
-  },
+  state = {currentSentByFilter: 'none'};
 
-  setSentByFilter(newFilter) {
-    this.setState({ currentSentByFilter: newFilter});
-  },
+  setSentByFilter = (currentSentByFilter) => this.setState({currentSentByFilter});
```

Comment:
are we cool with using (still-experimental) class properties for this and state?

Reply:
Yep!  We've been using the [Class properties transform](https://babeljs.io/docs/plugins/transform-class-properties/) for [six months now](https://github.com/code-dot-org/code-dot-org/pull/13589) and at least on starlabs we've been really into it.

Reply:
Awesome! Yeah, they're _so much_ cleaner than manual binding. I'mma start doing the same